docs/root/operations/admin.rst

```diff
@@ -350,6 +350,10 @@ modify different aspects of the server:
    This behaviour and duration is configurable via server options or CLI
    (:option:`--drain-time-s` and :option:`--drain-strategy`).
 
+   .. http:post:: /drain_listeners?graceful&skip_exit
+
+   When draining listeners, do not exit after the drain period. This must be used with `graceful`.
```

Comment:
We better to have an item for changelog https://github.com/envoyproxy/envoy/blob/a5930b27b083d71a92ca66ec5200cca65721f0a7/changelogs/current.yaml#L200